lbrynet/daemon/Daemon.py

```diff
@@ -97,6 +97,16 @@
 SHORT_ID_LEN = 20
 MAX_UPDATE_FEE_ESTIMATE = 0.3
 
+FILE_SORT_FIELD_NAME = 'name'
+FILE_SORT_FIELD_DATE = 'date'
+FILE_SORT_FIELD_PRICE = 'price'
+
+FILE_SORT_ORDER_ASCENDING = 'asc'
+FILE_SORT_ORDER_DESCENDING = 'desc'
+FILE_SORT_ORDERS = [
+    FILE_SORT_ORDER_ASCENDING,
+    FILE_SORT_ORDER_DESCENDING,
+]
```

Comment:
Could be a `tuple` instead of a `list`, conveys that this won't be modified elsewhere and takes up less memory.